services/frontend-service/src/ui/components/EnvironmentLane/EnvironmentLane.tsx

```diff
@@ -0,0 +1,47 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
+
+Copyright 2021 freiheit.com*/
+import { useFilteredEnvironmentLocks } from '../../utils/store';
+import { Button } from '../button';
+import { Delete } from '../../../images';
+import * as React from 'react';
+import { EnvironmentLockDisplay } from '../EnvironmentLockDisplay/EnvironmentLockDisplay';
+
+export const EnvironmentLane: React.FC<{ environment: string }> = (props) => {
```

Comment:
```suggestion
export const EnvironmentCard: React.FC<{ environment: string }> = (props) => {
```
I think Lane is not the best describe here

Reply:
Changed the name